components/tag/index.css

```diff
@@ -18,6 +18,7 @@ governing permissions and limitations under the License.
     filter: color;
     replace: --spectrum-tag$1;
   }
+  --spectrum-tag-texticon-text-font-size: var(--spectrum-global-dimension-font-size-75);
```

Comment:
I'm confused about this definition (and the ones on line `30` and `39`). Do we need to define these, or should we refer to what's coming across in the vars file? It looks like `--spectrum-tag-s-texticon-text-font-size` is defined in there.